core/src/main/java/org/renjin/primitives/Primitives.java

```diff
@@ -95,6 +96,17 @@ public static PrimitiveFunction getInternal(Symbol symbol) {
     return getPrimitive(INSTANCE.internalEntries, INSTANCE.internals, symbol);
   }
 
+  public static boolean isPrimitive(String opName) {
+    Symbol symbol = Symbol.get(opName);
+    return isPrimitive(INSTANCE.internalEntries, INSTANCE.internals, symbol);
```

Comment:
Why is this split out into two methods? Why are checking *both* the entries list and function cache? internalEntries is the definitive list. Why only internals and not builtins? Wouldn't `isPrimitive("+")` return `false`?